### ISSUE TITLE
Define default severity for each rule

### ISSUE THREAD

**Issue by nvuillam** [Force-Config-Control/lightning-flow-scanner-sfdx#55]
It seems that for the moment, when they are not overridden in config file, all rules are with **error** severity by default.

Some rules should just be warning, like missing flow description, because they are not as impacting than HardcodedId for example

It would be nice to have a default severity for each rule that is corresponding to its negative impact on the Salesforce org

**Comment by RubenHalman** [comment]
I really like this as Ive been thinking it will open up also creating a functionality to generate a report for prioritization in big projects. I think this should be combined with a rule category. In  most linters I know missing documentation actually gets the highest prio, so your given example would be implemented the other way around. How would we drive this discussion amongst our users to set the severity?

**Comment by nvuillam** [comment]
I think we can keep the "error", "warning" and "note" levels :)

Missing documentation being "note" as it's is not nice but can't break anything

As you are a flow expert, I suggest you make opiniotated decisions, then the community will make remarks if necessary :)

**Comment by nvuillam** [comment]
@RubenHalman in that case we should provide in documentation an example of config file with the full list of rules, so users can easily start from it instead of having to write it all ^^

**Comment by RubenHalman** [comment]
Yes good suggestion. I'll do that I the example repo, it's small so I'm keeping this closed if you don't mind.
